win32/src/winapi/mod.rs

```diff
@@ -56,11 +56,22 @@ pub const DLLS: [builtin::BuiltinDLL; 15] = [
     builtin::retrowin32_test::DLL,
 ];
 
+pub fn dll_alias(name: &str) -> Option<&'static str> {
+    Some(match name {
+        "msvcrt.dll" => "ucrtbase.dll",
```

Comment:
I guess these are not exactly the same, but one is a later version of the other?

Looking at wine they build two dlls but from the same sources, heh.  This should be fine.